airflow/models/taskinstance.py

```diff
@@ -1737,7 +1758,15 @@ def handle_failure(
 
         if error:
             if isinstance(error, BaseException):
-                self.log.error("Task failed with exception", exc_info=error)
+                execution_frame = _EXECUTION_FRAME_MAPPING.get(self.task)
```

Comment:
Is it possible that `self.task` is not in the dictionary?
```suggestion
                execution_frame = _EXECUTION_FRAME_MAPPING.get(self.task, None)
                if execution_frame: ...
```

Reply:
Doesn't get() return None if task is missing already :P ?

Reply:
🤦🏻 

For some reason I thought it would still error

Reply:
@ashb yes sometimes it is not in the dictionary, there are various ways a task can fail and only some of them will produce a traceback during execution.

Reply:
But yeah the `None` part is not needed – `get` should be fine if the task is not there with `None` being returned in that case.